app/jobs/address_proofing_job.rb

```diff
@@ -5,7 +5,8 @@ class AddressProofingJob < ApplicationJob
 
   discard_on JobHelpers::StaleJobHelper::StaleJobError
 
-  def perform(user_id:, issuer:, result_id:, encrypted_arguments:, trace_id:)
+  # rubocop:disable Lint/UnusedMethodArgument
+  def perform(issuer:, result_id:, encrypted_arguments:, trace_id:, user_id: nil)
```

Comment:
Yeah the two PRs got out of sync a bit as I was patching more issues

My current thinking is, maybe we don't want to remove the `user_id` as an argument? Since if we ever wanted to add analytics logging, it would be useful to have, etc